CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+* `FSUrlLoader#canLoad` reports false for local urls outside the loader's
```

Comment:
Why can't you load a file outside your root? Maybe add more context to the method name? `canLoadFromSomething`? Maybe I don't know what "load" means exactly.

Reply:
Well, it was a pre-existing behavior that `FSUrlLoader` restricted loads to its root.  What I've changed here is that `canLoad` used to always return `true` if the url given was a local file, regardless of whether in or out of the root.  However, `load` would throw an Error if it was outside the `root`.  This was lame because it meant we couldn't use a `MultiUrlLoader` with more than one `FSUrlLoader`.  I use the term `racking` in the sense of a `MultiUrlLoader` looking like a rack or stack of processors with fall-throughs; MultiUrlLoader resolved urls in sequence top-to-bottom, returning first defined result:
```
 |  InMemoryLoader 1
 |  FSUrlLoader 1
 |  FSUrlLoader 2
 V  InMemoryLoader 2
```

Maybe the word "racking" is obscure.  I could go with sequencing...

Reply:
...the reason we couldn't use them with MultiUrlLoader is the MultiUrlLoader couldn't interrogate the FSUrlLoader about whether a file load was possible.

Reply:
...and get a correct answer.

Reply:
Thanks for explaining!